### PR TITLE
Created a GMOD Service Option

### PR DESCRIPTION
**Pull request by Polarcraft**
This pull request will allow users to select Garry's Mod as a service option.

### CODE REVIEW

File: database/seeds/SourceServiceTableSeeder.php [hunk 1]
```diff
@@ -236,6 +236,50 @@ private function addCoreOptions()
...
+mkdir -p /mnt/server/steamcmd
+tar -xzvf steamcmd.tar.gz -C /mnt/server/steamcmd
+cd /mnt/server/steamcmd
+
+# SteamCMD fails otherwise for some reason, even running as root.
+# This is changed at the end of the install process anyways.
+chown -R root:root /mnt
+
+export HOME=/mnt/server
+./steamcmd.sh +login anonymous +force_install_dir /mnt/server +app_update 4020 +quit > /mnt/server/log.txt 2>&1
```
Comment (DaneEveritt) [inline database/seeds/SourceServiceTableSeeder.php]: Why is this piped into a log?

File: database/seeds/SourceServiceTableSeeder.php [hunk 2]
```diff
@@ -236,6 +236,50 @@ private function addCoreOptions()
...
+
+mkdir -p /mnt/server/.steam/sdk32 > /mnt/server/log2.txt 2>&1
```
Comment (DaneEveritt) [inline database/seeds/SourceServiceTableSeeder.php]: Same here?

File: database/seeds/SourceServiceTableSeeder.php [hunk 3]
```diff
@@ -236,6 +236,50 @@ private function addCoreOptions()
...
+cp -v linux32/steamclient.so ../.steam/sdk32/steamclient.so > /mnt/server/log3.txt 2>&1
```
Comment (DaneEveritt) [inline database/seeds/SourceServiceTableSeeder.php]: and here

File: database/seeds/SourceServiceTableSeeder.php (hunk 1 above)
Comment (Polarcraft) [inline database/seeds/SourceServiceTableSeeder.php]: I guess I didn't remove it from my testing, my bad.

File: database/seeds/SourceServiceTableSeeder.php (hunk 3 above)
Comment (Polarcraft) [inline database/seeds/SourceServiceTableSeeder.php]: I guess I didn't remove it from my testing, my bad.

File: database/seeds/SourceServiceTableSeeder.php (hunk 2 above)
Comment (Polarcraft) [inline database/seeds/SourceServiceTableSeeder.php]: I guess I didn't remove it from my testing, my bad.

**Pull request merged**